packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -224,6 +250,15 @@ export class QuoteRequestor {
                 );
                 return false;
             }
+            const minExpiry = Math.round((Date.now() + this._expiryBufferMs) / constants.ONE_SECOND_MS);
+            if (new BigNumber(response.expirationTimeSeconds).isLessThan(minExpiry)) {
```

Comment:
What do you think about putting this in a helper fn?

Reply:
Love it, done here: https://github.com/0xProject/0x-monorepo/pull/2574/commits/125c53827bbde841a0d978c405452a14008dffd6#diff-951722c8cfc47f2ea3eebf81342ae48dR318